ansible/roles/node/handlers/main.yml

```diff
@@ -3,6 +3,7 @@
   command: systemctl --system daemon-reload
   notify:
     - restart daemons
+  when: ansible_distribution != "Ubuntu"
```

Comment:
Is there some better test here?  Because ubuntu is getting systemd 'soon', right?  Does init system get collected in the setup facts?